core/domain/email_manager.py

```diff
@@ -55,6 +58,8 @@
 transaction_services = models.Registry.import_transaction_services()
 secrets_services = models.Registry.import_secrets_services()
 
+Registry = platform_parameter_registry.Registry
```

Comment:
PlatformParameterRegistry, perhaps (since you have a models.Registry just above, too)

Reply:
Done